SUPPORT.md

```diff
@@ -0,0 +1,28 @@
+# Support
+
+## Thoth wiki
+The Thoth team actively contributes to its [Wiki](../../wiki), if you have a question about book metadata we might have an entry about it.
+
+Our wiki also includes the [Thoth User Manual](../../wiki/Thoth-User-Manual).
+
+## Asking quality questions
+> This section has been borrowed from Titus Wormer's [remarkjs support document](https://github.com/remarkjs/.github/blob/main/support.md)
+
+Questions can go to [GitHub discussions][chat].
```

Comment:
Does this link go anywhere?

Reply:
Well spotted - I hadn't added the references below, so nope